cmd/govim/vimstate.go

```diff
@@ -89,6 +89,17 @@ type vimstate struct {
 	// lastProgressText points to the text in the most recently created progress
 	// popup.
 	lastProgressText *strings.Builder
+
+	// vimgrepPendingBufs contain buffers read during a vimgrep quickfix command,
```

Comment:
Can we perhaps link here to a Vim issue/discussion that answers whether this behaviour is expected or not?

Reply:
Done